client/store/selectors/rescues.js

```diff
@@ -1,8 +1,62 @@
+import { createSelector } from 'reselect'
+import { selectCurrentUserId, withCurrentUserId } from './session'
+import { selectGroupsByUserId } from './groups'
+import userHasPermission from '../../helpers/userHasPermission'
+
+
+
+// Constants
+const PAPERWORK_MAX_EDIT_TIME = 3600000
+
+
+
+
+
 const selectRescues = (state) => state.rescues.rescues
 
 const selectRescueById = (state, { rescueId }) => state.rescues[rescueId]
 
+const selectRescueCanEdit = createSelector(
+  [selectRescueById, selectCurrentUserId, withCurrentUserId(selectGroupsByUserId)],
+  (rescue, userId, userGroups) => {
+    if (!rescue || !userId) {
+      return false
+    }
+
+    // Check if current user is assigned to case.
+    const usersAssignedRats = rescue.relationships.rats.data?.reduce(
+      (acc, rat) => {
+        if (rat.attributes.userId === userId) {
+          return [...acc, rat]
+        }
+        return acc
+      }
+    )
+
+    if (usersAssignedRats.length) {
+      return true
```

Comment:
It should be User is Assigned AND rescue is less than an hour old, not OR

Reply:
huh. I though it was supposed to be or so that dispatchers could file 🤔

Reply:
erm this actually makes no sense... if only being able to edit if you're assigned AND within 1h of creation then A LOT of paperwork would never be able to be filed by normal rats :P